lib/mapreduce/mapreduce.go

```diff
@@ -527,3 +528,13 @@ func WaitJob(outRepo, branch, commit, job string) error {
 	log.Printf("WaitJob: %s %s %s %s", outRepo, branch, commit, job)
 	return btrfs.WaitForFile(path.Join(outRepo, branch, ".progress", commit, job))
 }
+
+func getDockerHost() (host string) {
+	host = Os.Getenv("DOCKER_HOST")
```

Comment:
Minor preference for calling this "PFS_DOCKER_HOST" just so that when it's seen outside the program it'll be immediately obvious who this environment variable is for.